python/py-freebase/Portfile

```diff
@@ -18,7 +18,7 @@ long_description    Freebase.com has an open API that can be used to \
                     http-based API is completely documented at \
                     Freebase.com along with some simple examples. This \
                     library provides a python interface to that API.
-homepage            https://code.google.com/p/freebase-python/
+homepage            https://developers.google.com/freebase
```

Comment:
Not commenting on your change ...

> Hey there! Are you maybe looking for Firebase instead?
> The Freebase API has been shut down. This page provides access to the last available data dump.

Reply:
Yes, I know - there are several python ports (and probably in other categories) for which there has been no upstream activity anymore in recent years. I think that everything we have in the ports tree should be up-to-date with upstream, however recent that might be.

Ideally, all these unmaintained, old ports could/should be removed, but whenever someone attempts that it leads to discussion and I don't want to go into that right now. At some point there was discussion about getting build/download statistics set-up (for some GSOC year if I remember correctly), which would be very helpful as it would allow us to remove ports that are hardly being installed anymore by end-users.